Integrate Vale for documentation (#4150)
pyvista/pyvista: doc/source/api/core/filters.rst

Composite Filters
~~~~~~~~~~~~~~~~~

> These are filters that can be applied to composite datasets, for example

Reviewer: If the original was correct then this should be
```suggestion
These are filters that can be applied to composite datasets, that is
```
